packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.scss

```diff
@@ -1,10 +1,30 @@
 @import '../../../../../fxa-content-server/app/styles/variables';
+@import '../../../../../fxa-content-server/app/styles/breakpoints';
 
 .product-payment {
+  background: #fff;
+  border: 1px solid #e7e7e7;
+  margin: 30px 0;
+  max-width: 600px;
+  min-width: 375px;
+  padding: 28px 16px 60px;
+
+  @include min-width('tablet') {
+    padding: 28px 48px 48px;
+  }
+
+  @include min-width('desktop') {
+    margin: 30px 24px;
```

Comment:
You see how ~ beautiful ~ it is going from mobile to tablet to desktop like this? 😇😂